vizro-core/docs/pages/user-guides/dashboard.md

```diff
@@ -210,6 +210,53 @@ You can provide [Markdown text](https://markdown-guide.readthedocs.io/) as a str
 
 ![Dashboard with tooltip](../../assets/user_guides/dashboard/dashboard_with_info_icon.png)
 
+## Customize the header
+
+You can now append custom content to the dashboard header using the [`custom_header`][vizro.models.Dashboard.custom_header] static method of the [Dashboard][vizro.models.Dashboard] class. This allows you to inject any Dash component(s) into a dedicated area of the header.
```

Comment:
Would it help to give examples of what "custom content" could be? Maybe just refer to what is shown in the image itself. I think now we are in a world of gen AI, we should use verbal clues to the models so they can repeat them, as they're not (AFAICT) going to be able to "look" at the graphic and see what the custom content is.